packages/main/src/types/ToolbarPriority.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Defines the priority of the toolbar item to go inside overflow popover.
+ *
+ * @readonly
+ * @enum {string}
+ * @public
+ * @type {string}
+ * @author SAP SE
+ * @alias sap.ui.webc.main.types.ToolbarPriority
+ */
+enum ToolbarPriority {
```

Comment:
I think this should be `ToolbarItemPriority` because it gets a bit confusing: `ToolbarDesign` and `ToolbarAlign` etc. refer to the toolbar itself, and then we get `ToolbarPriority`, which however pertains to the items, not the toolbar. I don't have a strong opinion though.